Gems/ROS2/Code/Source/RobotImporter/ROS2RobotImporterEditorSystemComponent.cpp

```diff
@@ -61,4 +75,92 @@ namespace ROS2
         options.toolbarIcon = ":/ROS2/ROS_import_icon.svg";
         AzToolsFramework::RegisterViewPane<RobotImporterWidget>("Robot Importer", "ROS2", options);
     }
+
+    bool ROS2RobotImporterEditorSystemComponent::GeneratePrefabFromFile(
+        const AZStd::string_view filePath, bool importAssetWithUrdf, bool useArticulation)
+    {
+        if (filePath.empty())
+        {
+            AZ_Warning("ROS2EditorSystemComponent", false, "Empty file path");
```

Comment:
`Empty file path` is a little vague, can we use 'Path provided for prefab is empty' ?
```suggestion
            AZ_Warning("ROS2EditorSystemComponent", false, "Path provided for prefab is empty");
```